image/tools/lib/component/resources.sh

```diff
@@ -134,6 +134,10 @@ function component_dump_data {
         backup_resource brokeredinfraconfigs ${ns} ${dest}
         backup_resource consoleservices ${ns} ${dest}
         backup_resource iotconfigs ${ns} ${dest}
+        backup_resource mobilesecurityservicedbs ${ns} ${dest}
```

Comment:
I do not think that we should do the backup of the mobilesecurityservices, mobilesecurityservicebackups and mobilesecurityservicedbs.

To restore the MSS Oper and its components the step could be re-run the installation steps.

Reply:
As per SOP PR, only app CRs will be used for restoration:
https://github.com/fheng/integreatly-help/pull/173/files#diff-a516303a301d4ba0827c1cb5b4f16297R331

I think there is nothing wrong with backing up other CRs. You never know when you might need to take a look at them. IMHO, in case of backups more is always better :D

Reply:
It is because:
- We/Admin should not use them
- Why store something that should not be used
- It can cause confusion
- It can cause other issues.

Reply:
We backup all created resources  by default, even if those can be created later by installed components  (fuse and amq online being examples of it).

Backing a resource up doesn't mean we need to restore all of them but it is usefull to archive those for auditing and backup restoration validation purposes.

Reply:
Thanks for the discussion guys. @camilamacedo86 based on the above comments I think we are okay to ahead with the implementation as it is.

Reply:
Yes. I am :-)